app/views/RoomView/index.tsx

```diff
@@ -656,11 +661,11 @@ class RoomView extends React.Component<IRoomViewProps, IRoomViewState> {
 		});
 	};
 
-	getPaddingLeft = (numIcons: number, isMasterDetail: boolean) => {
+	getPaddingRight = (numIcons: number, isMasterDetail: boolean) => {
 		if (numIcons === 3) {
-			return isMasterDetail ? 40 : 35;
+			return isMasterDetail ? 25 : 15;
```

Comment:
I tried a lot remove this calc, but I couldn't. Because of this white spaces

![Screen Shot 2022-06-24 at 00 29 06](https://user-images.githubusercontent.com/14894380/175456415-dfb98f4d-b791-4fa3-bb62-a97c4e4d8f8c.png)

the right botton don't get the third icon as part of it, the only way to  get more right space is decreasing the left space, but we have problem with tablets. So I removed the white space from the left and calcute the space from the third icon.

![Screen Shot 2022-06-24 at 00 32 55](https://user-images.githubusercontent.com/14894380/175456753-9043f4de-f808-4e14-b475-61718a657385.png)

this way will work fine for all platforms.